packages/ui/src/lib/modal/Modal.svelte

```diff
@@ -35,12 +35,13 @@ if (previously_focused) {
 <div class:items-center="{!top}" class="absolute w-full h-full flex justify-center">
   <button
     aria-label="close"
-    class="fixed top-0 left-0 w-full h-full bg-black bg-opacity-60 bg-blend-multiply z-40"
+    class="fixed top-0 left-0 w-full h-full bg-[var(--pd-modal-fade)] bg-blend-multiply z-40"
+    style="opacity: 0.6"
```

Comment:
It seems to work by using class `opacity-60` (instead of `bg-opacity-60`)

```suggestion
    class="fixed top-0 left-0 w-full h-full bg-[var(--pd-modal-fade)] bg-blend-multiply z-40 opacity-60"
```

Reply:
So focused on replicating what was there I failed to see the obvious. 🤦🏼‍♂️ Fixed.